### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by julzedz**
In this project, I did the following:
- Create a class Classroom.
- Create a class Book.
- Create a class Rental.
- Create the has-many/belongs-to relationships.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, book)
+    Rental.new(date, book, self)
+  end
```
Comment (julie-ify) [inline book.rb]: - [ ] We already have book through self, so we don't need to pass in book again, we should be passing in `person`.

- [ ] In your rental constructor, `person` comes before `book` in the arrangement of the arguments, so we should keep to that.

  ```
  def add_rental(date, person)
    Rental.new(date, person, self)
  end
  ```

File: person.rb [hunk 2]
```diff
@@ -20,6 +21,10 @@ def can_use_services?
     of_age? || parent_permission
   end
 
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```
Comment (julie-ify) [inline person.rb]: - [ ] We already have person through self, so we don't need to pass in person again, we should be passing in `book`.

- [ ] We should keep to the order of the arguments.

  ```
  def add_rental(date, book)
    Rental.new(date, self, book)
  end
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @julzedz,

Good job so far!

## Highlights!
- Good code organization and working associations ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @julzedz, 

To highlight 👍🏽 
✔️ Associations are created
✔️ Classes are created

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**